apps/src/weblab2/layout/HorizontalLayout.tsx

```diff
@@ -14,8 +16,11 @@ const MIN_EDITOR_HEIGHT = 200;
 const MIN_PREVIEW_HEIGHT = 200;
 const INITIAL_PREVIEW_HEIGHT = 400;
 const MIN_RIGHT_PANEL_WIDTH = 300;
+const PROJECT_FOOTER_HEIGHT = 64;
```

Comment:
In weblab2 horizontal layout, the footer content does not seem to be center-aligned vertically.
![Screenshot 2025-03-27 at 11 38 54 AM](https://github.com/user-attachments/assets/093fd0d8-6053-44fd-a18d-6a22c968587f)

I know `FilePreview` was a bit problematic with the resizing in the past.

Reply:
Weird, yours is a different height than mine (see screenshots...it might be because of hitting minimum sizes?). Is it center aligned with a larger screen size?

Reply:
But I am inclined generally to just file tickets for things like this in web lab 2, since we aren't dedicating resources to it right now.

Reply:
Oh yeah. If I make screen longer, it is aligned.
Comparing `pytnonlab` and `weblab` in screencast - `pythonlab` footer remains intact as expected.

https://github.com/user-attachments/assets/d75933da-aad6-42cb-bf5b-78392552a88e

Makes sense to file for follow-up work!

Reply:
https://codedotorg.atlassian.net/browse/CT-1163